src/components/MediaCrud/UploadAudio.js

```diff
@@ -31,8 +31,8 @@ function UploadAudio({ site, extensionList, setSelectedMedia }) {
     title: '',
     acknowledgement: '',
     description: '',
-    excludeFromGames: false,
-    excludeFromKids: false,
+    includeInGames: 'true',
+    includeInKids: 'true',
```

Comment:
Are these intentionally string values? I know js is weird and will count any string as truth-y, but if it's meant to be a boolean we should probably use a real one

Reply:
hmmm, it was so long ago, I don't remember why I made these strings. I'll change them back to booloeans